### PR TITLE
New kick area

### PR DESCRIPTION
**Pull request by Flova**
## Proposed changes
* Use overlapping thresholds for kick area
* Fix stuck head behavior

### CODE REVIEW

File: bitbots_head_behavior/src/bitbots_head_behavior/head_node.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def run(dsd):
 
     :returns: Never
     """
-    rate = Rate(125)
+    rate = Rate(60)
```
Comment (SammyRamone) [inline bitbots_head_behavior/src/bitbots_head_behavior/head_node.py:28]: why reduce the rate? higher rates mean smoother motion -> less forces acting on the robot -> less falling

Comment (Flova) [reply bitbots_head_behavior/src/bitbots_head_behavior/head_node.py:28]: But the head behavior doesn't include any interpolation and just sets the final goals. In certain conditions, it checks the path for collisions and this check gets very CPU heavy at such high rates. The dsd also doesn't include many sequences which might take multiple steps.

Comment (Flova) [reply bitbots_head_behavior/src/bitbots_head_behavior/head_node.py:28]: I also talked with Timon about this.

Comment (SammyRamone) [reply bitbots_head_behavior/src/bitbots_head_behavior/head_node.py:28]: ah okay then it makes sense. I thought that we interpolate

**Review by SammyRamone — APPROVED** [review]

File: bitbots_blackboard/src/bitbots_blackboard/capsules/head_capsule.py [hunk 2]
```diff
@@ -76,29 +77,65 @@ def send_motor_goals(self, pan_position, tilt_position, pan_speed=1.5, tilt_spee
...
+                return True
+        else: # Passes the stuff through
             self.pos_msg.positions = pan_position, tilt_position
             self.pos_msg.velocities = [pan_speed, tilt_speed]
             self.pos_msg.header.stamp = rospy.Time.now()
             self.position_publisher.publish(self.pos_msg)
             return True
 
+    def avoid_collision_on_path(self, goal_pan, goal_tilt, current_pan, current_tilt, pan_speed, tilt_speed, max_depth=4, depth=0):
+        # Backup behavior if mach recursion depth is reached
```
Comment (jgueldenstein) [inline bitbots_blackboard/src/bitbots_blackboard/capsules/head_capsule.py:95]:
```suggestion
        # Backup behavior if max recursion depth is reached
```

**Pull request merged**